### ISSUE TITLE
<Dropdown/> - no option is not marked when dropdown is opened

### ISSUE THREAD

**Issue by LittleTal** [wix/wix-style-react#3165]
# 🐛 Bug Report

### 🏗 Relevant Components
`<Dropdown/> `

...

### 😯 Current Behavior
When focused, dropdown layout is opened and no option is marked

![image](https://user-images.githubusercontent.com/23282301/54912475-5f0cac00-4ef9-11e9-8248-d259d6ff4092.png)

...

### 🤔 Expected Behavior
When opening dropdown layout, first option should be marked
![image](https://user-images.githubusercontent.com/23282301/54912531-79468a00-4ef9-11e9-98ad-c24ab5e440ff.png)

...

### 👣 Steps to Reproduce
Should be clear from the description

  - **WSR version**: `...`
  - **Yoshi version**: `...`

### 👀 Severity
Low

- Critical / Major / Low

**Comment by shlomitc** [comment]
Hey, thanks for reporting.
@mosheKerbel, I remember you did some a11y work in this area lately, was this the suggested behavior?

**Comment by mosheKerbel** [comment]
Yes, but if you take a look at Erez' work in the spec, it should be as a prop - with a default behaviour of false: markFirstMatchingOption
https://github.com/wix/wix-style-react/blob/master/src/MultiSelect/README.DESIGN-SPEC.md

**Comment by shlomitc** [comment]
Did Neil say it's better to have it marked or not?

**Comment by mosheKerbel** [comment]
We did not discuss it.
Lital was the one to mention it.

**Comment by shlomitc** [comment]
closing due to inactivity, we can re-open if relevant again
